src/containers/shared/components/TokenSearchResults/TokenSearchRow.tsx

```diff
@@ -0,0 +1,119 @@
+import { Link } from 'react-router-dom'
+import { useTranslation } from 'react-i18next'
+import { Amount } from '../Amount'
+import { localizeNumber } from '../../utils'
+import Currency from '../Currency'
+import DomainLink from '../DomainLink'
+
+const parsePrice = (dollarPrice: string, xrpPrice: number): number => {
+  const parsedDollar = Number(dollarPrice)
+  return Number((parsedDollar * xrpPrice).toFixed(6))
+}
+
+const renderLogo = (token) =>
```

Comment:
Use a React component style name

```suggestion
const TokenLogo = (token) =>
```